pymc_extras/statespace/core/statespace.py

```diff
@@ -1109,6 +1109,7 @@ def _sample_conditional(
         group: str,
         random_seed: RandomState | None = None,
         data: pt.TensorLike | None = None,
+        method: str = "svd",
```

Comment:
do you want to type hint the options and/or mention them on the docstrings? Potentially mention the reason for choosing some over the other?

Reply:
Just pushed this without having read this comment, so yes :)